config/initializers/constants.rb

```diff
@@ -0,0 +1,2 @@
+# Hold Google maps API key
+GOOGLE_MAPS_API_KEY = Rails.application.secrets.GOOGLE_MAPS_API_KEY
```

Comment:
Could you please add the newline here. I have actually had it cause issues.
In production
At night

Thanks for your help
Kevin